httpie/utils.py

```diff
@@ -12,10 +12,29 @@
 RE_COOKIE_SPLIT = re.compile(r', (?=[^ ;]+=)')
 
 
+class JsonDictWithDuplicateKeys(OrderedDict):
```

Comment:
Maybe instead of `with` something like `allowing` or `supporting`.

Reply:
…or actually`preserving` might the most fitting :)

Reply:
So it will be ;)